core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -175,7 +176,6 @@
     private MeterRegistry meterRegistry = Metrics.globalRegistry;
     private String serviceLoggerPrefix = DEFAULT_SERVICE_LOGGER_PREFIX;
     private AccessLogWriter accessLogWriter = AccessLogWriter.disabled();
-    private boolean shutdownAccessLogWriterOnStop = true;
```

Comment:
We had a chat discussion and decided to keep `shutdownOnStop` for `AccessLogWriters` for:

- consistency with other resources such as `blockingTaskExecutor()` and `workerGroup()`
- disambiguation of resource life cycle, because it is no longer clear if `AccessLogWriter` will be shut down on server stop by default.